nilearn/tests/test_extmath.py

```diff
@@ -50,3 +55,22 @@ def test_is_spd_with_symmetrical_matrix():
     matrix = np.array([[2, 1],
                        [1, 1]])
     assert is_spd(matrix, verbose=0)
+
+
+def test_fast_abs_percentile_no_index_error():
+    # Regression test
+    #####################
```

Comment:
Just curious about the human psyche, why do you feel the need to add your own pet markup that is not used anywhere else in the file?

Aside from that, I don't this comments adds a lot.